cmd/snappy/cmd_enable_classic.go

```diff
@@ -0,0 +1,84 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2014-2015 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/ubuntu-core/snappy/classic"
+	"github.com/ubuntu-core/snappy/i18n"
+	"github.com/ubuntu-core/snappy/logger"
+)
+
+type cmdEnableClassic struct {
+}
+
+var shortEnableClassicHelp = i18n.G("Enable classic dimension.")
+
+var longEnableClassicHelp = i18n.G("Enable the ubuntu classic dimension.")
+
+func init() {
+	_, err := parser.AddCommand("enable-classic",
```

Comment:
Regarding your comment about transforming this into snappy install classic, can we please talk about this before you push it forward?  I'm slightly concerned that this might become a leaky abstraction, so would appreciating understanding what depth of emulation you have in mind.